docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+version: '3.8'
+
+services:
+  nginx:
+    build: ./nginx
+    ports:
+      - 80:80
+    volumes:
+      - static_volume:/home/app/roboprop/static
+    depends_on:
+      - web
+    restart: "on-failure"
+  web:
+    build: . #build the image for the web service from the dockerfile in parent directory
+    command: sh -c "python manage.py makemigrations &&
+                    python manage.py migrate &&
+                    npx tailwindcss -i ./static/src/input.css -o ./static/src/output.css &&
+                    gunicorn roboprop.wsgi:application --bind 0.0.0.0:8000"
+    volumes:
+      - .:/roboprop:rw # map data and files from parent directory in host to roboprop directory in docker container
```

Comment:
is that were models, etc, would be saved?

Reply:
ah no, this would just be to map the roboprop source code to the container from the host, so if someone wants to make a change on the host machine it would be reflected in the container. Admittedly more useful for development than production perhaps.

models etc are saved on the fileserver, hosted elsewhere (this pr just for client)

Reply:
Ah indeed, I now realize there is no copy in the Dockerfile. so is this a Dockerfile for production or development?

Reply:
Copy statement is on L29 of Dockerfile.

This is meant primarily for production (nginx server). Remove the mapping to avoid confusion?

Reply:
oh indeed!
I think it is good as it is. Maybe a comment on the use case of this line then?